test/client/metadataApiRetrieve.test.ts

```diff
@@ -40,7 +40,7 @@ const messages = Messages.load('@salesforce/source-deploy-retrieve', 'sdr', [
   'error_no_components_to_retrieve',
 ]);
 
-describe('MetadataApiRetrieve', () => {
+describe.skip('MetadataApiRetrieve', () => {
```

Comment:
Should a comment be added above here about why this test is being skipped?

Reply:
Mike had a PR for some testSetup fix that's in now.  It's re-enabled.